packages/Sandblocks-Core/SBForceConnector.class.st

```diff
@@ -0,0 +1,70 @@
+Class {
+	#name : #SBForceConnector,
+	#superclass : #SBConnectorLine,
+	#category : #'Sandblocks-Core'
+}
+
+{ #category : #'as yet unclassified' }
+SBForceConnector >> baseForce [
+	
+	^ 0.1 * (self to center - self from center) 
+]
+
+{ #category : #'as yet unclassified' }
+SBForceConnector >> color [
+
+	 ^ self to foregroundColor
+]
+
+{ #category : #'as yet unclassified' }
+SBForceConnector >> connect: aMorph to: anotherMorph [
+
+	super connect: aMorph to: anotherMorph.
+	
+	aMorph withDecorator: SBForceMoveDecorator do: [:decorator | decorator registerAdditionalForce: self].
+	anotherMorph withDecorator: SBForceMoveDecorator do: [:decorator | decorator registerAdditionalForce: self]
+]
+
+{ #category : #'as yet unclassified' }
+SBForceConnector >> connectorType [
+
+	 ^ #none
+]
+
+{ #category : #'as yet unclassified' }
+SBForceConnector >> deleteBlock [
+	
+	self abandon
+]
+
+{ #category : #'as yet unclassified' }
+SBForceConnector >> forceFor: aMorph [
+	
+	^ (aMorph == self from) 
```

Comment:
klammern optional :)

Reply:
also, it's rather untypical to use `==` as you block fancy things like Proxies from working. Just using `=` will also do `==` in the end for a Morph, unless it's wrapped in a proxy, in which case it would forward it to the nested object's `==`.